### PR TITLE
Renable encryption for linux container environment

### PR DESCRIPTION
**Pull request by balag0**

### CODE REVIEW

File: src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj [hunk 1]
```diff
@@ -57,7 +57,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="3.0.11" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Logging" Version="3.0.11" />
 	  <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.3" />
-    <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.88-alpha" />
+    <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.999-alpha" />
```
Comment (balag0) [inline src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj]: This version is from staging feed. Once https://github.com/Azure/azure-websites-security/pull/9 is merged this should be updated with the new version from production feed.

File: src/WebJobs.Script.WebHost/Security/KeyManagement/DefaultKeyValueConverterFactory.cs [hunk 2]
```diff
@@ -23,12 +18,6 @@ public DefaultKeyValueConverterFactory(bool allowEncryption)
 
         private static bool IsEncryptionSupported()
         {
-            if (SystemEnvironment.Instance.IsLinuxContainerEnvironment())
```
Comment (mathewc) [inline src/WebJobs.Script.WebHost/Security/KeyManagement/DefaultKeyValueConverterFactory.cs]: We want to include this IsLinuxContainerEnvironment check below, so we're doing IsAppServiceEnvironment || IsLinuxContainerEnvironment

**Comment by mathewc** [comment]
Replacing with modified PR https://github.com/Azure/azure-websites-security/pull/10

**Comment by mathewc** [comment]
Replaced by https://github.com/Azure/azure-functions-host/pull/4759

**Pull request closed without merging**